icicle-compiler/test/Icicle/Test/Sea/Seaworthy.hs

```diff
@@ -34,7 +35,7 @@ namer = A.namerText (flip Var 0)
 --   Like prop_psv, but without psv.
 prop_seaworthy wt
  = testIO
- $ do let Just a = asAttributeName "eval"
+ $ do let a = [inputid|default:eval|]
```

Comment:
so there are some places where a namespace is required, and I think this is one of them